### PR TITLE
DOCS-6660-added semantic versioning details

### PR DESCRIPTION
**Pull request by priyaayyar**

### CODE REVIEW

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc [hunk 1]
```diff
@@ -81,7 +81,7 @@ To create an API using Anypoint Service Mesh, perform the following steps:
 | `environmentId` | The ID of the environment where you want to create your API instance. | Required | Not applicable
 | `groupId` | The ID of the group of assets on Exchange, for example, the organization ID. You can find the organization ID on API Manager landing page or in Access Management. | Optional | Organization ID of the organization that contains the environment.
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc]:
```suggestion
| `groupId` | The ID of the group of assets on Exchange, for example, the organization ID. You can find the organization ID on the API Manager landing page or in Access Management. | Optional | Organization ID of the organization that contains the environment. 
```

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc [hunk 2]
```diff
@@ -81,7 +81,7 @@ To create an API using Anypoint Service Mesh, perform the following steps:
 | `environmentId` | The ID of the environment where you want to create your API instance. | Required | Not applicable
 | `groupId` | The ID of the group of assets on Exchange, for example, the organization ID. You can find the organization ID on API Manager landing page or in Access Management. | Optional | Organization ID of the organization that contains the environment. 
 | `assetId` | The ID of the assets that are a part of an organization. Supports alphanumeric characters without spaces. | Required | Not applicable
-| `version` | The version of the asset on Exchange. *Needs to be semver* what is this chirag? | Required | Not applicable
+| `version` | The version of the asset on Exchange. The version number must follow the semantic versioning format for major, minor, and patch releases. | Required | Not applicable
 | `instanceLabel` | The ID of the API instances of an Asset within a given environment. Supports alphanumeric characters without spaces. | Required | Not applicable
 | `specTags` | Tags to be added to the asset on Exchange. |  Optional | Not applicable
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc]:
```suggestion
| `specTags` | Tags to add to the asset on Exchange. |  Optional | Not applicable
```

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc [hunk 3]
```diff
@@ -81,7 +81,7 @@ To create an API using Anypoint Service Mesh, perform the following steps:
...
 | `instanceTags` | Tags to be added to the API instance on API Manager. |  Optional | Not applicable
```
Comment (valkyrie69) [inline modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc]:
```suggestion
| `instanceTags` | Tags to add to the API instance on API Manager. |  Optional | Not applicable
```

**Review by valkyrie69 — APPROVED** [review]

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc (hunk 1 above)
Comment (priyaayyar) [reply modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc]: Done! Thanks!

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc]: Done! Thanks!

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc]: Done! Thanks!

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc [hunk 4]
```diff
@@ -79,12 +79,12 @@ To create an API using Anypoint Service Mesh, perform the following steps:
 | `name` | The name of the API that you want to provision, for example, `orders-api`. | Required | Not applicable
 | `namespace` | The namespace in which you want to create the adapter, for example, `mulesoft-apps`. | Optional | default
 | `environmentId` | The ID of the environment where you want to create your API instance. | Required | Not applicable
-| `groupId` | The ID of the group of assets on Exchange, for example, the organization ID. You can find the organization ID on API Manager landing page or in Access Management. | Optional | Organization ID of the organization that contains the environment. 
+| `groupId` | The ID of the group of assets on Exchange, for example, the organization ID. You can find the organization ID on the API Manager landing page or in Access Management. | Optional | Organization ID of the organization that contains the environment.
```
Comment (chiragnarangmulesoft) [inline modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc:82]: @tbruno Is org Id really optional ? 
> | Optional | Organization ID of the organization that contains the environment.

File: modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc [hunk 5]
```diff
@@ -79,12 +79,12 @@ To create an API using Anypoint Service Mesh, perform the following steps:
...
 | `environmentId` | The ID of the environment where you want to create your API instance. | Required | Not applicable
-| `groupId` | The ID of the group of assets on Exchange, for example, the organization ID. You can find the organization ID on API Manager landing page or in Access Management. | Optional | Organization ID of the organization that contains the environment. 
+| `groupId` | The ID of the group of assets on Exchange, for example, the organization ID. You can find the organization ID on the API Manager landing page or in Access Management. | Optional | Organization ID of the organization that contains the environment. 
 | `assetId` | The ID of the assets that are a part of an organization. Supports alphanumeric characters without spaces. | Required | Not applicable
-| `version` | The version of the asset on Exchange. *Needs to be semver* what is this chirag? | Required | Not applicable
+| `version` | The version of the asset on Exchange. The version number must follow the semantic versioning format for major, minor, and patch releases. | Required | Not applicable
```
Comment (chiragnarangmulesoft) [inline modules/ROOT/pages/create-an-api-configure-service-mesh-CLI.adoc:84]: Add an eg - 1.0.0 or 1.0.1

**Pull request merged**